dbms/src/Storages/DeltaMerge/File/DMFileReader.cpp

```diff
@@ -540,31 +541,44 @@ void DMFileReader::readFromDisk(
     bool force_seek)
 {
     const auto stream_name = DMFile::getFileNameBase(column_define.id);
-    if (auto iter = column_streams.find(stream_name); iter != column_streams.end())
-    {
-        auto & top_stream = iter->second;
-        bool should_seek = force_seek || shouldSeek(start_pack_id) || skip_packs > 0;
-        auto data_type = dmfile->getColumnStat(column_define.id).type;
-        data_type->deserializeBinaryBulkWithMultipleStreams( //
-            *column,
-            [&](const IDataType::SubstreamPath & substream_path) {
-                const auto substream_name = DMFile::getFileNameBase(column_define.id, substream_path);
-                auto & sub_stream = column_streams.at(substream_name);
-
-                if (should_seek)
-                {
-                    sub_stream->buf->seek(
-                        sub_stream->getOffsetInFile(start_pack_id),
-                        sub_stream->getOffsetInDecompressedBlock(start_pack_id));
-                }
-                return sub_stream->buf.get();
-            },
-            read_rows,
-            top_stream->avg_size_hint,
-            true,
-            {});
-        IDataType::updateAvgValueSizeHint(*column, top_stream->avg_size_hint);
-    }
+    auto iter = column_streams.find(stream_name);
+#ifndef NDEBUG
+    RUNTIME_CHECK_MSG(
+        iter != column_streams.end(),
+        "Can not find column_stream, column_id={} stream_name={}",
+        column_define.id,
+        stream_name);
+
+#endif
```

Comment:
Why not just use `assert`?

Reply:
assert in here don't give any information about the column, using RUNTIME_CHECK helps us locate the bug more quickly under tests.